charts/eap-xp4/README.md

```diff
@@ -60,30 +60,30 @@ The configuration for the image that is built and deployed is configured in a `i
+| `build.s2i.jdk11.runtimeImage` | EAP S2I Runtime image for JDK 11 | `registry.redhat.io/jboss-eap-7/eap-xp4-openjdk11-runtime-openshift-rhel8` | [EAP Documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/getting_started_with_jboss_eap_for_openshift_container_platform/index)                                                                                                                                                               |
+| `build.s2i.galleonLayers` | A list of layer names to compose a EAP server | - | [EAP Documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/getting_started_with_jboss_eap_for_openshift_container_platform/index)                                                                                                                                                               |
+ | `build.s2i.galleonDir` | Directory relative to the root directory for the build that contains custom content for Galleon. | - | [EAP Documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/getting_started_with_jboss_eap_for_openshift_container_platform/index)                                                                                                                                                               |
  | `build.s2i.featurePacks` | List of additional Galleon feature-packs identified by Maven coordinates (`<groupId>:<artifactId>:<version>`) | - | The value can be be either a `string` with a list of comma-separated Maven coordinate or an array where each item is the Maven coordinate of a feature pack - [EAP Documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/getting_started_with_jboss_eap_for_openshift_container_platform/index) |
-| `build.bootableJar.builderImage` | JDK Builder image for Bootable Jar | [registry.access.redhat.com/ubi8/openjdk-11:latest](https://catalog.redhat.com/software/containers/ubi8/openjdk-11/5dd6a4b45a13461646f677f4?gti-tabs=unauthenticated) | - |
+| `build.bootableJar.builderImage` | JDK Builder image for Bootable Jar | [registry.access.redhat.com/ubi8/openjdk-11:latest](https://catalog.redhat.com/software/containers/ubi8/openjdk-11/5dd6a4b45a13461646f677f4?gti-tabs=unauthenticated) | -                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
It seems there's a formatting issue on your IDE that's adding spaces after end on line?

Reply:
Yes, I am using intellij idea ultimate latest version. But whenever I am trying to put a space or special character like comma after  " Allowed Values: `"11"` ", a new line gets generated in the table. Although, I can confirm that the preview of the table looks fine to me. 
I am still trying to remove this formatting issue and will send an updated change soon.

Reply:
You can format it using Ctrl+Alt+L or use a plugin [pipe-table-formatter](https://plugins.jetbrains.com/plugin/16355-pipe-table-formatter) and For trailing spaces use Settings → Editor → General → Strip trailing spaces

Reply:
Thanks @RanabirChakraborty . Although, these didn't work; I had to manually change it in vim.